spec-tests/src/main/java/org/candlepin/spec/bootstrap/data/builder/Products.java

```diff
@@ -43,7 +43,7 @@ public static ProductDTO random() {
      *  a product DTO with a randomly generated SKU ID and name
      */
     public static ProductDTO randomSKU() {
-        String id = String.valueOf((int) Math.random() * 100000);
+        String id = String.valueOf((int) (Math.random() * 100000));
```

Comment:
I was getting an id value of 0 and I think the issue was that the double returned from Math.random() was being cast to an integer before multiplying by 100,000 and so it was rounding up to 0 and causing the 0 * 100,000 operation to result in 0.